### PR TITLE
fixed 5 errors

### PR DESCRIPTION
**Pull request by Anantdeep**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml [hunk 1]
```diff
@@ -47,32 +47,32 @@ skin.sheet.addTemplate=Add a new overriding template
 skin.sheet.add=Add
 skin.sheet.overrideTemplateWarning=This section allows you to override a template of the skin. This is technical and you should be aware of the consequences.
 skin.sheet.deprecatedPropertiesWarning=This section is here for retro-compatibility reasons. You should use the 'overridden templates' mechanism above.
-skin.sheet.nojs=Without JavaScript, you cannot use this section. But you can use the {0}Object Editor{1} instead, and manually add {2} objects.
+skin.sheet.nojs=Without JavaScript, you cannot use this section,but you can use the {0}Object Editor{1} instead, and manually add {2} objects.
```
Comment (akshitdewan) [inline xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml]: you should add a space after the comma

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml]: Personally I prefer short sentences. If the issue is starting a sentence with "But" then an easy solution is to replace it with "However". You might as well rewrite the sentence in this case... "You cannot use this section without Javascript enabled. However you could use the {0}Object Editor{1}, and manually add {2} objects."

File: xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml [hunk 2]
```diff
@@ -47,32 +47,32 @@ skin.sheet.addTemplate=Add a new overriding template
...
 skin.sheet.js.addingDone=The template has been added!
 
 ###############################################################################
 ## Deprecated
-## Note: each element should be removed when the last branch using it is no longer supported
+## Note: each element should be removed when the last branch using it is no longer supported.
 ###############################################################################
 
-## Used to indicate where deprecated keys start
+## Used to indicate where deprecated keys start,
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml]: This is not a translation.

File: xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml [hunk 3]
```diff
@@ -47,32 +47,32 @@ skin.sheet.addTemplate=Add a new overriding template
...
-skin.sheet.js.deleteConfirmationNo=No, cancel
+skin.sheet.js.deleteConfirmationYes=Yes,delete it
+skin.sheet.js.deleteConfirmationNo=No,cancel
 skin.sheet.js.adding=Adding...
 skin.sheet.js.addingDone=The template has been added!
 
 ###############################################################################
 ## Deprecated
-## Note: each element should be removed when the last branch using it is no longer supported
+## Note: each element should be removed when the last branch using it is no longer supported.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml]: This is not a translation.

File: xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml [hunk 4]
```diff
@@ -47,32 +47,32 @@ skin.sheet.addTemplate=Add a new overriding template
...
+## Used to indicate where deprecated keys start,
 #@deprecatedstart
 
 #######################################
 ## until 7.1M1
 #######################################
 skin.sheet.overridedTemplates=Overridden templates
 
-## Used to indicate where deprecated keys end
+## Used to indicate where deprecated keys end,
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml]: This is not a translation.

**Pull request closed without merging**